sites/partners/__tests__/PaperListingForm/LotteryResults.test.tsx

```diff
@@ -0,0 +1,161 @@
+import React from "react"
+import { fireEvent, render } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { rest } from "msw"
+import { setupServer } from "msw/node"
+import axios from "axios"
+import LotteryResults from "../../src/listings/PaperListingForm/sections/LotteryResults"
+import { FormProvider, useForm } from "react-hook-form"
+import { formDefaults, FormListing } from "../../src/listings/PaperListingForm/formTypes"
+import { ListingEventType, serviceOptions } from "@bloom-housing/backend-core"
+
+const FormComponent = ({ children, values }: { values?: FormListing; children }) => {
+  const formMethods = useForm<FormListing>({
+    defaultValues: { ...formDefaults, ...values },
+    shouldUnregister: false,
+  })
+  return <FormProvider {...formMethods}>{children}</FormProvider>
+}
+
+const server = setupServer()
+
+// Enable API mocking before tests.
+beforeAll(() => {
+  serviceOptions.axios = axios.create({
+    baseURL: "http://localhost:3100",
+  })
+  server.listen()
+})
+
+// Reset any runtime request handlers we may add during the tests.
+afterEach(() => server.resetHandlers())
+
+// Disable API mocking after the tests are done.
+afterAll(() => server.close())
+
+describe("LotteryResults", () => {
+  it("Should not render anything when drawerState is false", () => {
+    // Need to set __next on base div to handle the overlay
+    const portalRoot = document.createElement("div")
+    portalRoot.setAttribute("id", "__next")
+    document.body.appendChild(portalRoot)
+
+    const submitFn = jest.fn()
+    const showDrawerFn = jest.fn()
+    const results = render(
+      <FormComponent>
+        <LotteryResults submitCallback={submitFn} drawerState={false} showDrawer={showDrawerFn} />
+      </FormComponent>
+    )
+
+    expect(results.container.innerHTML).toEqual("")
+  })
+
+  it("Should Render Lottery Results in open state", () => {
+    // Need to set __next on base div to handle the overlay
+    const portalRoot = document.createElement("div")
```

Comment:
I'd love to make this a shared global jest util in `setup-tests` or testHelpers if possible, but could definitely happen not in this PR

Reply:
+1

Reply:
Great Idea! Moved